tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -110,6 +110,8 @@ function install_artifacts() {
 	chmod +x /opt/kata/bin/*
 	[ -d /opt/kata/runtime-rs/bin ] && \
 		chmod +x /opt/kata/runtime-rs/bin/*
+    # Best effort to set selinux context on host
+    nsenter --target 1 --mount bash -c "chcon -Rt usr_t /opt/kata/ && chcon -t bin_t /opt/kata/bin/* && chcon -t bin_t /opt/kata/libexec/* && echo 'selinux labeles set successfully for /opt/kata'; chcon -t bin_t /opt/kata/runtime-rs/bin/*" || true
```

Comment:
nit - `s/labeles/labels`
fwiw, I use a similar approach to deploy latest upstream shim on OpenShift for testing.

Reply:
Well this PR is seriously outdated, we use:

```yaml
apiVersion: apps/v1
kind: DaemonSet
metadata:
  name: relabel-selinux-daemonset
  namespace: confidential-containers-system
spec:
  selector:
    matchLabels:
      app: restorecon
  template:
    metadata:
      labels:
        app: restorecon
    spec:
      serviceAccountName: cc-operator-controller-manager
      hostPID: true
      containers:
        - name: relabel-selinux-container
          image: alpine
          securityContext:
            privileged: true
          command: ["/bin/sh", "-c", "
            set -e;
            echo Starting the relabel;
            nsenter --target 1 --mount bash -xc '
                command -v semanage &>/dev/null || { echo Does not look like a SELINUX cluster, skipping; exit 0; };
                for ENTRY in \
                    \"/(.*/)?opt/kata/bin(/.*)?\" \
                    \"/(.*/)?opt/kata/runtime-rs/bin(/.*)?\" \
                    \"/(.*/)?opt/kata/share/kata-.*(/.*)?(/.*)?\" \
                    \"/(.*/)?opt/kata/share/ovmf(/.*)?\" \
                    \"/(.*/)?opt/kata/share/tdvf(/.*)?\" \
                    \"/(.*/)?opt/kata/libexec(/.*)?\";
                do
                    semanage fcontext -a -t qemu_exec_t \"$ENTRY\" || semanage fcontext -m -t qemu_exec_t \"$ENTRY\" || { echo \"Error in semanage co
mmand\"; exit 1; }
                done;
                restorecon -v -R /opt/kata || { echo \"Error in restorecon command\"; exit 1; }
            ';
            echo NSENTER_FINISHED_WITH: $?;
            sleep infinity"]
```

in the upstream OCP testing. Before updating this PR I'm waiting for a green light from the community on whether we want to include the selinux handling.

Reply:
basically here we'd use the `semanage` instead of the `chcon` which should be a bit clearer (especially for long-term deployments)